config/config.php

```diff
@@ -2,203 +2,285 @@
+    |          'mutation' => [
+    |
+    |          ],
+    |          'middleware' => ['auth'],
+    |      ],
+    |      'user/me' => [
+    |          'query' => [
+    |              'profile' => 'App\GraphQL\Queries\MyProfileQuery'
+    |          ],
+    |          'mutation' => [
+    |
+    |          ],
+    |          'middleware' => ['auth'],
+    |      ],
+    |  ]
+    |
+    */
+
     'schemas' => [
+
         'default' => [
-            'query' => [
-                // 'example_query' => ExampleQuery::class,
-            ],
-            'mutation' => [
-                // 'example_mutation'  => ExampleMutation::class,
-            ],
+
+            'query' => [],
+            'mutation' => [],
             'middleware' => [],
-            'method'     => ['get', 'post'],
+            'method' => ['get', 'post'],
+
         ],
-    ],
 
-    // The types available in the application. You can then access it from the
-    // facade like this: GraphQL::type('user')
-    //
-    // Example:
-    //
-    // 'types' => [
-    //     'user' => 'App\GraphQL\Type\UserType'
-    // ]
-    //
-    'types' => [
-        // 'example'           => ExampleType::class,
-        // 'relation_example'  => ExampleRelationType::class,
     ],
 
-    // This callable will be passed the Error object for each errors GraphQL catch.
-    // The method should return an array representing the error.
-    // Typically:
-    // [
-    //     'message' => '',
-    //     'locations' => []
-    // ]
+    /*
+    |--------------------------------------------------------------------------
+    | The types available in the application. You can then access it from the
+    | facade like this: GraphQL::type('user')
+    |--------------------------------------------------------------------------
+    |
+    | Example:
+    |
+    | 'types' => [
+    |     'user' => 'App\GraphQL\Types\UserType'
+    | ]
+    |
+    */
+
+    'types' => [],
+
+    /*
+    |--------------------------------------------------------------------------
+    | This callable will be passed the Error object for each errors GraphQL
+    | catch. The method should return an array representing the error
+    |--------------------------------------------------------------------------
+    |
+    | Typically:
+    | [
+    |     'message' => '',
+    |     'locations' => []
+    | ]
+    |
+    */
+
     'error_formatter' => ['\Rebing\GraphQL\GraphQL', 'formatError'],
 
     /*
-     * Custom Error Handling
-     *
-     * Expected handler signature is: function (array $errors, callable $formatter): array
-     *
-     * The default handler will pass exceptions to laravel Error Handling mechanism
-     */
+    |--------------------------------------------------------------------------
+    | Custom Error Handling
+    |--------------------------------------------------------------------------
+    |
+    | Expected handler signature is:
+    | function (array $errors, callable $formatter): array
```

Comment:
I almost missed the signature function here; maybe surround with empty lines (and intend?)?